### PR TITLE
Adapt repository to Quarkus 2.2.3.Final

### PR DESCRIPTION
**Pull request by Sgitario**

### CODE REVIEW

**Comment by Sgitario** [comment]
cc @rsvoboda

**Comment by Sgitario** [comment]
cc @cescoffier

File: quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java [hunk 1]
```diff
@@ -45,15 +45,15 @@
     void shouldPingOpenAPI() {
         given()
             .header(ACCEPT, APPLICATION_JSON)
-            .when().get("/openapi")
+            .when().get("/q/openapi")
```
Comment (aureamunoz) [inline quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:48]: That should be done also for `hero` and `fight` microservices

File: quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java [hunk 2]
```diff
@@ -45,15 +45,15 @@
...
             .then()
             .statusCode(OK.getStatusCode());
     }
 
     @Test
     void shouldPingSwaggerUI() {
         given()
-            .when().get("/swagger-ui")
+            .when().get("/q/swagger-ui")
```
Comment (aureamunoz) [inline quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:56]: That should be done also for `hero` and `fight` microservices

File: quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java [hunk 3]
```diff
@@ -62,15 +62,15 @@ void shouldPingSwaggerUI() {
     @Test
     void shouldPingLiveness() {
         given()
-            .when().get("/health/live")
+            .when().get("/q/health/live")
```
Comment (aureamunoz) [inline quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:65]: That should be done also for `hero` and `fight` microservices

**Review by aureamunoz — CHANGES_REQUESTED** [review]
Dockerfiles should also be adapted.
I'm working on a [mirror of this repo](https://github.com/Red-Hat-Developer-Games/quarkus-workshop) and can do it later today (I thought to migrate the whole workshop once finished with the mirror :-))

File: quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java (hunk 3 above)
Comment (aureamunoz) [reply quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:65]: This needs also be reported to the html pages...

File: quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java (hunk 2 above)
Comment (aureamunoz) [reply quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:56]: This needs also be reported to the html pages...

File: quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java (hunk 1 above)
Comment (aureamunoz) [reply quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:48]: This needs also be reported to the html pages...

Comment (Sgitario) [reply quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:48]: Done

File: quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java (hunk 3 above)
Comment (Sgitario) [reply quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java:65]: Done

**Comment by Sgitario** [comment]
> Dockerfiles should also be adapted.
> I'm working on a [mirror of this repo](https://github.com/Red-Hat-Developer-Games/quarkus-workshop) and can do it later today or tomorrow (I thought to migrate the whole workshop once finished with the mirror :-))
> @Sgitario

The only missing bits are the Dockerfile files. However, if you prefer migrate this repo after you finished with the mirror, I'm happy to close this pull request!

**Comment by aureamunoz** [comment]
> > Dockerfiles should also be adapted.
> > I'm working on a [mirror of this repo](https://github.com/Red-Hat-Developer-Games/quarkus-workshop) and can do it later today or tomorrow (I thought to migrate the whole workshop once finished with the mirror :-))
> > @Sgitario
> 
> The only missing bits are the Dockerfile files. However, if you prefer migrate this repo after you finished with the mirror, I'm happy to close this pull request!

Ok, I will do it. I approved this one :-)

**Comment by cescoffier** [comment]
I was about to do these changes! Thanks!

**Review by aureamunoz — APPROVED** [review]
LGTM. Thanks @Sgitario !!

**Pull request merged**